README.adoc

```diff
@@ -83,16 +68,16 @@ fn main() {
         .parse::<SocketAddr>()
         .expect("Expect to parse address");
 
-    let client_addr = "127.0.0.1:8545"
-        .parse::<SocketAddr>()
-        .expect("Expect to parse client address");
-
-    emerald::start(&addr, &client_addr, None);
+    emerald::start(&addr, None);
```

Comment:
s/`emerald::start(&addr, None);`/`emerald::start(&addr, None, None);`/

That would fit the current fn definition.